tests/components/canary/test_sensor.py

```diff
@@ -184,7 +185,7 @@ def test_battery_sensor(self):
         sensor.update()
 
         assert "Home Family Room Battery" == sensor.name
-        assert "%" == sensor.unit_of_measurement
+        assert UNIT_PERCENTAGE == sensor.unit_of_measurement
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)